### PR TITLE
gyro_lpf to profile value

### PR DESCRIPTION
**Pull request by borisbstyle**
Not sure if it's needer, but some were requesting for this.

It's tested and it works

### CODE REVIEW

**Comment by joshuabardwell** [comment]
Now that I am wholeheartedly sold on soft filters, I don't really care, but before I fully transitioned, I wanted to have profile 1 be gyro_lpf=42 and profile 2 be soft filters. I couldn't do that with gyro_lpf being global.

**Comment by borisbstyle** [comment]
@joshuabardwell 
I don't see it as needed as well anymore, but here the code for those who want to test. Did receive a lot ot questions lately about this.
I just know that these people are only looking for easy way to test.

File: src/main/main.c [hunk 1]
```diff
@@ -363,7 +363,7 @@ void init(void)
     }
 #endif
 
-    if (!sensorsAutodetect(&masterConfig.sensorAlignmentConfig, masterConfig.gyro_lpf, masterConfig.acc_hardware, masterConfig.mag_hardware, currentProfile->mag_declination)) {
+    if (!sensorsAutodetect(&masterConfig.sensorAlignmentConfig, currentProfile->pidProfile.gyro_lpf, masterConfig.acc_hardware, masterConfig.mag_hardware, currentProfile->mag_declination)) {
```
Comment (ledvinap) [inline src/main/main.c]: This is called only on once boot so reboot is needed after changing profile. OK for testing, but it should be documented

**Comment by digitalentity** [comment]
@borisbstyle gyro_lpf is used only when initializing gyros on boot. Since profiles can be switch in flight, I think profiles should only hold settings that can be changed on the fly and do not require a reboot to take effect.

**Comment by borisbstyle** [comment]
and thats another good reason not to nake it a profile value :)

**Pull request closed without merging**